app/assets/javascripts/admin/enterprises/controllers/enterprise_controller.js.coffee

```diff
@@ -22,24 +22,26 @@ angular.module("admin.enterprises")
       $window.location = destination
 
     $scope.removeLogo = ->
-      Enterprises.removeLogo($scope.Enterprise).then (data) ->
-        $scope.Enterprise = angular.copy(data)
-        $scope.$emit('enterprise:updated', $scope.Enterprise)
+      if confirm(t('admin.enterprises.remove_logo.immediate_removal_warning'))
```

Comment:
you can say I am crazy and ignore but I always prefer less indentation:
if !confirm return
do_what_the_method_is_supposed_to_do_with_no_indentation